client/src/main/java/io/lionweb/client/LionWebClient.java

```diff
@@ -276,28 +276,28 @@ public LionWebVersion getLionWebVersion() {
   @Override
   public RepositoryVersionToken createPartitionsFromChunk(
       @NotNull List<SerializedClassifierInstance> data) throws IOException {
-    throw new UnsupportedOperationException();
+    return this.chunkLevelBulkAPIs.createPartitionsFromChunk(data);
```

Comment:
These methods (of `ClientForChunkLevelBulkAPIs`) are now hooked up, but I don't see them being unit tested.